src/kiwixapp.cpp

```diff
@@ -380,8 +380,7 @@ void KiwixApp::createAction()
     HIDE_ACTION(SettingAction);
 
     CREATE_ACTION_ICON(DonateAction, "donate", tr("Donate to support Kiwix"));
-    //SET_SHORTCUT(DonateAction, QKeySequence(Qt::CTRL+Qt::Key_BracketLeft+Qt::Key_3));
-    HIDE_ACTION(DonateAction);
+    SET_SHORTCUT(DonateAction, QKeySequence("Ctrl+<,3"));
```

Comment:
Is this shortcut really works ?

I cannot enter it. (But I'm in bepo on my computer)

Reply:
It's the keys "Ctrl+<" pressed together and then "3".
 I can modify it to be "Ctrl+<, Ctrl+3"

Reply:
Hum.. pretty sure that no one will be able to enter it correctly.
The `<3` is nice because it makes a :heart: but I think this shortcut is useless.

Let's move on with this. I'm merging the PR